docs/databases_klifs_statement_of_need.rst

```diff
@@ -0,0 +1,86 @@
+Statement of need
+=================
+
+
+
+`OpenCADD-KLIFS <https://opencadd.readthedocs.io/en/latest/databases_klifs.html>`_ 
+(``opencadd.databases.klifs``) is a part of the `OpenCADD <https://opencadd.readthedocs.io/>`_ 
+package, a collection of Python modules for structural cheminformatics.
```

Comment:
Maybe adding a short sentence about the general target audience in the first paragraph? (suggested) 
That statement will help to identify potential users.

Reply:
Good point, thanks! I added a short sentence about the general target audience at the very beginning and moved the more technical sentence shown above to the end of that paragraph.

Should I update the respective paper paragraph accordingly?

Reply:
Yes, please!